EKF/control.cpp

```diff
@@ -305,7 +305,10 @@ void Ekf::controlExternalVisionFusion()
 					_ev_pos_innov(1) = _state.pos(1) - _hpos_pred_prev(1) - ev_delta_pos(1);
 
 					// observation 1-STD error, incremental pos observation is expected to have more uncertainty
-					ev_pos_obs_var(0) = ev_pos_obs_var(1) = sq(fmaxf(_ev_sample_delayed.posErr, 0.5f));
+					Matrix3f ev_pos_var = matrix::diag(_ev_sample_delayed.posVar);
```

Comment:
Why not simply

`Vector3f ev_pos_var = _ev_rot_mat * _ev_sample_delayed.posVar`?

Reply:
OK I guess it's nicer to have it as a matrix if we ever end up actually propagating the covariances.